app/controllers/customer_resources_controller.rb

```diff
@@ -14,7 +14,9 @@ def show
   def update
     @customer_resource.update customer_resource_params
 
-    render status: :no_content
+    # re-fetch from RM API to surface side-effects
+    @customer_resource = customer_resources.find customer_resource_id
```

Comment:
Should this be handled at the model layer? e.g.

```ruby
@customer_resource = customer_resources.update customer_resource_id, customer_resourc_params
```

Reply:
Re-fetch moved to model layer.